scripts/swap-package-links.js

```diff
@@ -0,0 +1,39 @@
+const fse = require('fs-extra');
+const path = require('path');
+const {
+  displayConfirmationMessage,
+  displayErrorMessage
+} = require('./common');
+
+const [/* node bin */, /* path to this script */, pkgName] = process.argv;
+
+const PACKAGE_LOCATIONS = {
+  handsontable: './tmp',
+  '@handsontable/angular': './wrappers/angular-handsontable/dist/hot-table'
+};
+let packageList = {};
+
+if (pkgName) {
+  packageList[pkgName] = PACKAGE_LOCATIONS[pkgName];
+
+} else {
+  packageList = PACKAGE_LOCATIONS;
+}
+
+for (const [packageName, packageLocation] of Object.entries(packageList)) {
+  if (fse.pathExistsSync(`${packageLocation}`)) {
+    fse.removeSync(
+      path.resolve(`./node_modules/${packageName}`),
+    );
+
+    fse.ensureSymlinkSync(
```

Comment:
On bare clone, after `npm i` I've got symlink errors:
![image](https://user-images.githubusercontent.com/571316/105999855-7812b500-60ae-11eb-8d5d-067f6dc4239d.png)

Reply:
That's expected, as the source directories don't exist yet. I agree, that it might be misleading, I'll try to communicate it better.
The linking script is triggered after `install` + after handsontable's `build:es` and angular's `build`, so after the package directories are generated, the linking should be applied/reapplied.